packages/transit-vehicle-overlay/src/components/popups/vehicle-tracker.js

```diff
@@ -38,3 +38,7 @@ VehicleTracker.defaultProps = {
   vehicle: null,
   isTracked: false
 };
+
+// Rename styled components for export
+const Styled = S;
```

Comment:
I would `import * as Styled` and remove this statement and comment above.

Reply:
See previous comments, have updated the export style